src/components/Header.vue

```diff
@@ -13,7 +13,7 @@
             <a class="nav-link" href="#"><small>{{ $t("firstNavbar.aboutRoyalLibrary") | capitalize({ onlyFirstLetter: true }) }}</small></a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#" @click.stop="switchLanguage">
+            <a class="nav-link" href="#" @click.stop="toggleLanguage">
```

Comment:
Er der brug for en stop propagation her? Kan sagtens være du er rendt ind i noget jeg lige har overset.

Reply:
lavet om til v:on-click